app/controllers/api/base_controller.rb

```diff
@@ -107,6 +107,15 @@ def initialize
     end
 
     before_action :parse_api_request, :log_api_request, :validate_api_request, :validate_api_action
+    before_action :validate_response_format, :only => [:show]
```

Comment:
@abellotti is async done via POST? I can't seem to find the documentation on that. If so, would

```ruby
before_action :validate_response_format, :except => [:destroy]
```
Work for you?